app/controllers/statistics_controller.rb

```diff
@@ -0,0 +1,34 @@
+class StatisticsController < ApplicationController 
+
+  def index
+    @cars = current_user.cars.order(created_at: :asc)  
+
+    @total_repairs = current_user.cars.joins(:repairs).group_by_month(:date, format: "%B %Y").sum(:amount_cents)  
+
+    @total_chargings = current_user.cars.joins(:chargings).group_by_month(:date, format: "%B %Y").sum(:amount_cents)  
```

Comment:
you can create a scope in the model for such big queries

Reply:
It's done now. It's great someone to tell you about new things 😊. I'll not merge that branch to main yet because we made an agreement with Peter that till Monday we will do all issues, , so when he merge Taxes I'll add them here and then will merge them to main, also we will do maybe some unit testing too😒